goose3/configuration.py

```diff
@@ -102,6 +103,9 @@ def __init__(self):
 
         # set the local storage path
         # make this configurable
+        self._local_storage_path = None
+
+        # set the default by calling the setter
         self.local_storage_path = os.path.join(tempfile.gettempdir(), 'goose')
```

Comment:
isn't this defined as a `@property` function below?

Reply:
It is, it could be removed or modified to be the following which should be identical functionality.
``` python
self.local_storage_path = None
````

Reply:
You are completely correct, removed the duplicate declaration.